code/modules/clothing/under/color.dm

```diff
@@ -56,163 +67,133 @@
 /obj/item/clothing/under/color/grey
 	name = "grey jumpsuit"
 	desc = "A tasteful grey jumpsuit that reminds you of the good old days."
-	icon_state = "grey"
-	inhand_icon_state = "gy_suit"
+	greyscale_colors = "#b3b3b3"
 
 /obj/item/clothing/under/color/jumpskirt/grey
 	name = "grey jumpskirt"
 	desc = "A tasteful grey jumpskirt that reminds you of the good old days."
-	icon_state = "grey_skirt"
-	inhand_icon_state = "gy_suit"
+	greyscale_colors = "#b3b3b3"
 
 /obj/item/clothing/under/color/grey/ancient
 	name = "ancient jumpsuit"
 	desc = "A terribly ragged and frayed grey jumpsuit. It looks like it hasn't been washed in over a decade."
 	icon_state = "grey_ancient"
+	inhand_icon_state = "gy_suit"
 	can_adjust = FALSE
```

Comment:
this inherits greyscale_colors and config doesnt it? rainbow jumpsuit inherits config too

Reply:
Yep, it does, but due to no colors it does nothing at all

Reply:
Decided to make a more clear declaration that prevents config inheritance